newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesController.php

```diff
@@ -339,9 +339,11 @@ public function relatedArticlesAction(Request $request, $number, $language = nul
             $ids[]  = $relatedArticle->getArticleNumber();
         }
 
+        $articleSearchCriteria = new ArticleSearchCriteria();
+        $articleSearchCriteria->language = $article->getLanguage()->getCode();
         $articles = $em->getRepository('Newscoop\Entity\Article')
             ->getArticlesByIds(
```

Comment:
well, there is a function called `getArticlesByIds` in article repository which, yeah, makes no sense, will change the function name as it is used only in this controller